apps/meteor/tests/e2e/page-objects/omnichannel-units.ts

```diff
@@ -51,13 +51,13 @@ export class OmnichannelUnits extends OmnichannelAdministration {
 		await this.inputDepartments.click();
 		await this.inputDepartments.fill(name);
 		await this.findOption(name).click();
-		await this.contextualBar.click({ position: { x: 0, y: 0 } });
+		await this.contextualBar.click({ position: { x: 3, y: 3 } });
```

Comment:
why?

Reply:
The UI glitches a little bit when we click on the resizable boundary. Happens only on test env, causing the test to fail. I updated the coordinates to not click on the boundary.